src/components/Settings/SettingsAddressbook.vue

```diff
@@ -222,24 +222,27 @@ export default {
 				}
 			}, 500)
 		},
-		copyLink() {
+		copyLink(event) {
 			// change to loading status
 			this.copyLoading = true
+			event.stopPropagation()
+
 			// copy link for addressbook to clipboard
 			this.$copyText(this.addressbook.url).then(e => {
+				event.preventDefault()
 				this.copySuccess = true
 				this.copied = true
+				// timeout sets the text back to copy to show text was copied
+				setTimeout(() => {
```

Comment:
Hum,timeout are bad feedback when they're not relevant. Here the string is copied right away but we get the feeling we need to wait :)

Reply:
Ok I just thought otherwise it really doesn't show as its pretty quick!